_includes/nav.html

```diff
@@ -21,8 +21,9 @@
                 <li class="dropdown">
                     <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><svg aria-hidden="true" class="github-mark" height="22" version="1.1" viewBox="0 0 16 16" width="32"><path fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0 0 16 8c0-4.42-3.58-8-8-8z"></path></svg>Github <span class="caret"></span></a>
                     <ul class="dropdown-menu">
+                        <li><a target="_blank" href="https://github.com/interledger">Specs and Website</a></li>
                         <li><a target="_blank" href="https://github.com/interledgerjs">JS Reference Implementations</a></li>
-                        <li><a target="_blank" href="https://github.com/interledger">Specs and other repos</a></li>
+                        <li><a target="_blank" href="https://github.com/hyperledger/quilt">Java Reference Implementations</a></li>
```

Comment:
I think I would say just "Java Implementations". Arguably the JS implementation is the "reference implementation" now. I'd change the terminology once the protocols are stable and fully implemented by the different implementations

Reply:
Just want to be consistent with the JS item above, which also uses the wording  'Reference Implementations" so if we change this, let's change the other for consistency.

Reply:
It's different though. A reference implementation is the main one that other people should look at when they're trying to figure out how ILP works or are looking for something to base their implementation on. Right now the JS implementation is much more complete than the Java one, so I would call the JS implementation the reference and the Java another implementation.

Reply:
How about:

- Specs and Website
- InterledgerJS Javascript Reference Implementations
- Hyperledger Quilt Java Implementations

Reply:
👍